src/Bicep.Core/Parsing/ParamsParser.cs

```diff
@@ -74,13 +75,42 @@ public override ProgramSyntax Program()
         private UsingDeclarationSyntax UsingDeclaration()
         {
             var keyword = ExpectKeyword(LanguageConstants.UsingKeyword);
+
+            SyntaxBase expression = reader.Peek().Type switch
+            {
+                TokenType.EndOfFile or
+                TokenType.NewLine or
```

Comment:
Why would we treat EOF or newline as if the user has input `none`? Shouldn't this just be a syntax error?

Reply:
that's true, removed unneeded tokens from the check